src/CoreWCF.Primitives/src/Resources/Strings.resx

```diff
@@ -8217,4 +8217,7 @@
   <data name="SynchronousUserNamePasswordValidationIsDeprecated" xml:space="preserve">
     <value>Implementers should implement ValidateAsync instead.</value>
   </data>
+  <data name="UnknownServiceConfiguration" xml:space="preserve">
+    <value>Could not find service '{0}' in configured services. Service must be registered using 'builder.AddService&lt;TService&gt;(...)' before registering a service endpoint using 'builder.AddServiceEndpoint&lt;TService, TContract&gt;(...)'</value>
+  </data>
```

Comment:
Can you change the error message to refer to the IServiceBuilder interface instead of a presumed variable name?

Reply:
done!